src/sentry/receivers/outbox/region.py

```diff
@@ -58,7 +73,8 @@ def process_organization_member_updates(
             )
         return
 
-    organizationmember_mapping_service.create_with_organization_member(org_member=org_member)
+    # TODO: replace with organizationmember_mapping_service.update_with_organization_member(org_member=org_member)
```

Comment:
:+1:

Reply:
@markstory I'm adding this in https://github.com/getsentry/sentry/pull/47669.